Bricks/strings/printf.h

```diff
@@ -33,6 +33,9 @@ SOFTWARE.
 namespace current {
 namespace strings {
 
+#ifndef CURRENT_WINDOWS
```

Comment:
Hate to sound like a prick, but there are other compilers. Perhaps introduce a top-level `CURRENT_GCC_OR_CLANG` into `port.h` and guard by it here?